packages/cds-odata-annotation-converter/.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": [
+        "../../.eslintrc"
+    ],
+    "parserOptions": {
+        "EXPERIMENTAL_useSourceOfProjectReferenceRedirect": true,
+        "project": "./tsconfig-test.json"
```

Comment:
Please use the same `tscconfig.json` set up as other packages. See https://github.com/SAP/open-ux-tools/pull/1577.

Reply:
done